apps/indexer/lib/indexer/prometheus/block_instrumenter.ex

```diff
@@ -61,4 +62,8 @@ defmodule Indexer.Prometheus.BlockInstrumenter do
   def handle_set_event([:indexer, :blocks, :last_block_number], %{value: val}, _metadata, _config) do
     Gauge.set([name: :indexer_blocks_last_block_number_current], val)
   end
+
+  def handle_set_event([:indexer, :blocks, :pending_blockcount], %{value: val}, _metadata, _config) do
+    Gauge.set([name: :indexer_blocks_pending_blockcount_current], val)
```

Comment:
Nit: drop the current? or append count?
Since this is a timeseries everything will have a timestamp.

Reply:
I'd like to, but this requires a larger change as our current convention is to dynamically create metric ids whereby we append `_current` for `Gauge` events and `_total` for `Counter` events.

Ultimately I would change the general approach to allow more nuance in names.

Reply:
Actually I'll do it anyway

Reply:
Ah, I was missing context - whatever goes in the direction of uniformity works for me. Thanks for the details!
Changing all the metric names will require more effort and we'll need to be careful with dashboards and alerting.